css-brochure-form/styles.css

```diff
@@ -0,0 +1,53 @@
+h1 {
+  text-align: center;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: 20px;
+  font-weight: 550;
+  color: #fe6053;
+}
+
+div {
+  margin-bottom: 10px;
+}
+
+label {
+  margin-left: 513px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: bold;
+  font-size: 12px;
+}
+
+select {
+  margin-left: 500px;
+  width: 438px;
+  height: 41px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  background-color: #edede9;
+  border-radius: 5px;
+  border-color: #ced4da;
+}
+
+.form-style {
+  margin-left: 500px;
+  width: 438px;
+  height: 40px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: 17px;
+  border-radius: 5px;
+  border: #ced4da solid 1px;
+}
+
+#submit-bttn {
+  margin-left: 837px;
```

Comment:
instead of using all this margin, I recommend wrapping the entire page in a `container` element, and centering that on the page using margin auto.

Reply:
![CSS-Brochure](https://user-images.githubusercontent.com/90743100/169706152-eca2dd07-4854-4511-8ec1-2121bec09ab5.png)